.gitattributes

```diff
@@ -0,0 +1,3 @@
+*.ts text eol=crlf
+*.js text eol=crlf

```

Comment:
Moving this into the linter will inconvenient because it will require the developers to change the environment settings. I believe this approach is superior because it lets git auto-convert the line endings on a commit